<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultipleChat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_root">
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widgetSidebar">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_sidebar">
        <item>
         <widget class="QFrame" name="frameProfile">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_profile">
           <item>
            <widget class="QLabel" name="lblAvatar">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>头像</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_profileText">
             <item>
              <widget class="QLabel" name="lblNickname">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>未登录</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblStatus">
               <property name="text">
                <string>离线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_profile">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSearch">
          <property name="placeholderText">
           <string>搜索好友/群组</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabNav">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabContacts">
           <attribute name="title">
            <string>好友</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_contacts">
            <item>
             <widget class="QTreeWidget" name="treeContacts">
              <property name="headerHidden">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabGroups">
           <attribute name="title">
            <string>群组</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_groups">
            <item>
             <widget class="QListWidget" name="listGroups"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabRecent">
           <attribute name="title">
            <string>最近</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_recent">
            <item>
             <widget class="QListWidget" name="listRecent"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_sidebarButtons">
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCreateGroup">
            <property name="text">
             <string>建群</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSettings">
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetContent">
       <layout class="QVBoxLayout" name="verticalLayout_content">
        <item>
         <widget class="QLabel" name="lblConversationTitle">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>群聊</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="txtChatHistory">
          <property name="placeholderText">
           <string>聊天记录</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_input">
          <item>
           <widget class="QTextEdit" name="txtMessageInput">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>70</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>输入消息，回车发送（后续可实现）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSend">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxLog">
          <property name="title">
           <string>运行日志</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_log">
           <item>
            <widget class="QTextBrowser" name="txtClientLog"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
